pdfmaker/css/_modules/base.scss

```diff
@@ -1111,6 +1111,24 @@ p[class*="ExtractSource"] {
   orphans: 2;
 }
 
+p[class*="Extract-Noteextn"] {
+	text-indent: none;
+	text-align: center;
+	margin: $gridheight;
+	hyphens: none;
+}
+
+span.spanaltfont1span1 {
+  	font-family: FeltpenCom-Medium;
+  	font-size: $sizehwritingprint;
+  	line-height: $gridheight;
```

Comment:
Did you find that it was necessary to add a line-height definition to get these spans to look right? It isn't typical to add line-height defs to inline elements--it's usually used only for block elements, and the span will inherit it from the block.